TCMSApp/src/client/app/defects/defects.html

```diff
@@ -1,3 +1,10 @@
+<style>
+
+.chosenDefect{
```

Comment:
it can be moved to our styles.less file and the color should be moved into color variable there (or we can use some predefined color if there is one suitable). and better class name is actually .selected-row, and here is why:
- don't use case sensitive class names, it will be used in case-insensitive html
- we will use this class not only for defects so that our UI could look nice and consistent 

Maybe there is something like selected-row in bootstrap? :hammer: